stix_shifter/stix_transmission/src/modules/proxy/proxy_connector.py

```diff
@@ -0,0 +1,46 @@
+from ..base.base_connector import BaseConnector
+import json, requests
+
+class Connector(BaseConnector):
+    def __init__(self, connection, configuration):
+
+        self.is_async = True
+
+        self.connection = connection
+        self.configuration = configuration
+
+        self.results_connector = self
+        self.status_connector = self
+        self.delete_connector = self
+        self.query_connector = self
+        self.ping_connector = self
+
+    def ping(self):
+        response = requests.post("http://" + self.connection["proxy_host"] + ":" + self.connection["proxy_port"] + "/ping",
+            data=json.dumps({"connection":self.connection,"configuration":self.configuration}))
+        return response.text
+
+    def ping(self):
```

Comment:
Duplicate ping methods.

Reply:
You're reviewing an old commit; see latest PR

Reply:
I am looking at the latest.